control-plane/src/modules/runs/agent/tools/stdlib.ts

```diff
@@ -0,0 +1,22 @@
+import { buildCalculatorTool } from "./calculator";
+import { buildCurrentDateTimeTool } from "./date-time";
+import { buildGetUrlTool } from "./get-url";
+
+// TODO: this should be Record<string, string> after we consolidate on AgentToolV2
+export const stdlib = {
+  calculator: {
```

Comment:
Nice Just remember to add these to 
https://docs.inferable.ai/pages/standard-lib